ui/app/templates/allocations/allocation/task/index.hbs

```diff
@@ -30,8 +30,8 @@
       </div>
     </div>
   {{/if}}
-  <h1 class="title with-flex" data-test-title>
-    <div>
+  <Hds::PageHeader class="job-page-header" as |PH|>
```

Comment:
Making use of our new [Page Header component](https://helios.hashicorp.design/components/page-header)